### PR TITLE
Add timestamp field to Comment class and a few improvements

### PR DESCRIPTION
**Pull request by PaulKovalov**
Here I add a timestamp field to the comment class. I haven't done it yesterday because I was not sure what was the best way to implement it. So this is the implementation I think is fine. I also change the way of how serializer copies `Comment` objects fields.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/CommentRepresentationSerializer.java [hunk 1]
```diff
@@ -1,20 +1,33 @@
...
+      try {
+        String fieldName = field.getName();
+        String fieldValue = (String)field.get(comment);
+        if (fieldValue != null) {
+          CommentRepresentationSerializer.class.getField(fieldName).set(this, fieldValue);
+        }
+      } catch (IllegalAccessException | NoSuchFieldException ex) {
+        System.out.println(ex.getMessage());
+      }
+    }
```
Comment (NeilLi1992) [inline portfolio/src/main/java/com/google/sps/servlets/CommentRepresentationSerializer.java:31]: Why do we need reflection for construction after adding the new field `timestamp`, but not before?

Comment (PaulKovalov) [reply portfolio/src/main/java/com/google/sps/servlets/CommentRepresentationSerializer.java:31]: We need it in general, I just wrote it here.

Comment (PaulKovalov) [reply portfolio/src/main/java/com/google/sps/servlets/CommentRepresentationSerializer.java:31]: I mean the idea that I need some generic way to serialize all `Comment`'s fields came to me when I was adding timestamp.

Comment (PaulKovalov) [reply portfolio/src/main/java/com/google/sps/servlets/CommentRepresentationSerializer.java]: Improved it using inheritance

**Pull request merged**